SoftLayer/managers/vs.py

```diff
@@ -465,7 +465,7 @@ def reload_instance(self, instance_id,
                                 'FORCE', config, id=instance_id)
 
     def _generate_create_dict(
-            self, cpus=None, memory=None, hourly=True,
+            self, cpus=None, memory=None, useHourlyPricing=True,
```

Comment:
Don't change names of function arguments if we can help it. Change this back to "hourly" so we don't break anyone's code that happens to be using this function.